lib/engine-test.js

```diff
@@ -52,7 +52,7 @@ EngineTest.prototype.getVersion = function(callback) {
 
   var result = '';
   var flag = ENGINE_TEST_ARGS[this.type] || '-v';
-  var version = childProcess.spawn(this.type, [flag]);
+  var version = crossSpawn.spawn(this.type, [flag]);
```

Comment:
Looks like cross-spawn will attempt to use the native child_process "shell" option if the flag is passed and the node version supports it. I don't have a Windows machine with me at the moment but I'll try it out this evening.